package.json

```diff
@@ -87,60 +87,60 @@
     "bip39": "^3.0.4",
     "bip65": "^1.0.3",
     "bip68": "^1.0.4",
-    "bitcoinjs-lib": "^6.0.0",
+    "bitcoinjs-lib": "^6.0.1",
     "blessed": "^0.1.81",
     "clightningjs": "^0.2.2",
-    "commander": "^6.1.0",
+    "commander": "^9.3.0",
     "content-type": "^1.0.4",
-    "dotparser": "^0.3.0",
+    "dotparser": "^1.1.1",
     "ecpair": "^2.0.1",
     "elliptic": "^6.5.4",
-    "fast-json-patch": "^2.0.7",
+    "fast-json-patch": "^3.1.1",
     "fd-chunk-store": "^2.0.0",
-    "glob-run": "^0.1.6",
+    "glob-run": "^0.1.7",
     "javascript-state-machine": "^3.1.0",
-    "jayson": "^3.4.4",
+    "jayson": "^3.6.6",
     "js-beautify": "^1.14.3",
-    "json-pointer": "^0.6.0",
-    "level": "^7.0.1",
+    "json-pointer": "^0.6.2",
+    "level": "^8.0.0",
     "lodash.merge": "^4.6.2",
-    "merkletreejs": "^0.1.6",
+    "merkletreejs": "^0.2.31",
     "minsc": "^0.2.0",
     "mkdirp": "^1.0.4",
-    "p-event": "^4.2.0",
+    "p-event": "^5.0.1",
     "path-match": "^1.2.4",
     "pluralize": "^8.0.0",
-    "redis": "^3.1.2",
-    "rimraf": "^2.6.2",
+    "redis": "^4.1.0",
+    "rimraf": "^3.0.2",
     "simple-aes": "^0.1.1",
     "struct": "^0.0.12",
     "tiny-secp256k1": "^2.2.1",
-    "ts-morph": "^10.0.2"
+    "ts-morph": "^15.1.0"
   },
   "devDependencies": {
-    "chai": "^4.0.2",
-    "codecov": "^3.6.1",
-    "cross-env": "^5.1.3",
-    "debug-trace": "^2.2.1",
-    "docco": "^0.7.0",
-    "eccrypto": "^1.0.3",
-    "eslint": "^5.16.0",
+    "chai": "^4.3.6",
+    "codecov": "^3.8.2",
+    "cross-env": "^7.0.3",
+    "debug-trace": "^2.2.3",
+    "docco": "^0.9.1",
+    "eccrypto": "^1.1.6",
+    "eslint": "^8.17.0",
     "gitbook-cli": "^2.3.2",
     "gitbook-plugin-edit-link": "^2.0.2",
-    "gitbook-plugin-exercises": "^3.0.0",
-    "gitbook-plugin-page-toc": "^1.1.0",
+    "gitbook-plugin-exercises": "^1.0.0",
```

Comment:
Is this a downgrade?  I think gitbook might one of the candidates for removal; they went fairly commercial and stopped supporting self-hosted GitBooks (@ ~v3).